<?xml version="1.0" ?>
<sdf version="1.5">
  <model name="mpnet_book">
    <pose frame="world">1 0 0.0 0 0 0</pose>
    <static>true</static>
    <link name="link">
      <inertial>
        <mass>0.1</mass>
      </inertial>
      <visual name="visual">
        <geometry>
          <mesh>
            <uri>model://mpnet_book/meshes/Dictionary.STL</uri>
            <scale>0.0017 0.0017 0.0017</scale>
          </mesh>
        </geometry>
      </visual>
    </link>
  </model>
</sdf>
